package.json

```diff
@@ -17,7 +17,9 @@
     "concurrently": "^5.3.0",
     "less": "^3.12.2",
     "less-watch-compiler": "^1.14.6",
+    "lessc": "^1.0.2",
     "onchange": "^7.0.2",
     "prettier": "^2.1.1"
-  }
+  },
+  "dependencies": {}
```

Comment:
Please manually remove this line as it is not important anymore.

Reply:
ok